src/Fable.Cli/Pipeline.fs

```diff
@@ -68,6 +70,62 @@ module Js =
     }
 
 module Python =
+
+    let encodeString charRequiresEncoding (value: string) : string =
```

Comment:
@alfonsogarciacaro I've reimpl `JavaScriptStringEncode` from HttpEncoder.cs. We could probably reuse this function for the other languages as well and just have separate `charRequiresEncoding` for each. What do you think?

Reply:
That's a great idea! :+1: We can also add a note to say where it comes from. Maybe move it to Prelude.fs (Naming module) where we have the other polyfills?